go/controller/api/v1alpha1/autogenagent_types.go

```diff
@@ -31,6 +31,8 @@ type AgentSpec struct {
 	Description string `json:"description,omitempty"`
 	// +kubebuilder:validation:MinLength=1
 	SystemMessage string `json:"systemMessage,omitempty"`
+	// +kubebuilder:validation:Required
+	ModelConfigRef string `json:"modelConfigRef"`
```

Comment:
Why is this required now? Currently we support a default model config which feels like a nice default UX